data-complexity/fisher_discriminant_ratio.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+from dcm import dcm
+from sklearn.datasets import load_iris
+
+
+def fisher_discriminant_ratio(features: pd.DataFrame, target: pd.DataFrame, ) -> float:
```

Comment:
I would rename the function to `f1`. Shortened version should be more convenient to use in code.